html_reference.html

```diff
@@ -67,8 +68,10 @@ <h3 class="container">portfolio  <i class="fas fa-palette" id="palette"></i></h2
 				<a><i class="fas fa-share-alt-square"></i></a>
```

Comment:
Can't comment on it directly, but the HTML validator is complaining that buttons aren't allowed to have the href attribute. Consider replacing them with anchor tags.

Reply:
changed!